build/AzurePipelinesTemplates/WindowsAppSDK-BuildInstaller-Stage.yml

```diff
@@ -0,0 +1,29 @@
+# This Installer Stage is purely for validation
+# The actual installer is built with the Aggregator build
+stages:
+- stage: InstallerValidation
+  jobs:
+  - job: BuildInstaller
+    dependsOn: []
+    pool:
+      type: windows
+      isCustom: true
+      name: 'ProjectReunionESPool-2022' # This stage is purely for validation
+    strategy:
+      maxParallel: 10
+      matrix:
+        Release_x86:
+          buildPlatform: 'x86'
+          buildConfiguration: 'Release'
+        Release_x64:
+          buildPlatform: 'x64'
+          buildConfiguration: 'Release'
+        Release_arm64:
+          buildPlatform: 'ARM64'
```

Comment:
NIT: Why ARM64 and not arm64 like the other architectures?

Reply:
Good point. I will change and see if everything still builds